nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/JoltTransformJSON.java

```diff
@@ -130,6 +130,15 @@
             .required(true)
             .build();
 
+    public static final PropertyDescriptor PRETTY_PRINT = new PropertyDescriptor.Builder()
+            .name(("Pretty Print"))
```

Comment:
There's an extra set of parentheses here, and the `name` field is usually formatted like `pretty_print` because it is used in XML object resolution. The `displayName` field below is used for the human-readable and changeable value that won't affect object resolution during flow loading.

Reply:
Pushed an update, thanks! While we are looking at that property, is there any reason that the TRANSFORM_CACHE_SIZE property (Just above pretty print) is the only one in this processor that isn't 'public'?

Reply:
It looks like it was added by a different author, and there are no external references to it, so I believe it can be changed to `private`. @markap14 and @YolandaMDavis , do you have any reasons it should not be changed? Thanks. 

If the change is made, it should also have a `displayName` added for consistency.